test/unit/postprocessing/baseline/README.md

```diff
@@ -0,0 +1,7 @@
+To regenerate the baseline plots run the following command from the root directory of the repository
```

Comment:
Thanks for including this. Does it belong in CONTRIBUTING, though?

Reply:
I honestly don't know. At some point CONTRIBUTING becomes too large and people won't read it anymore. I'd rather have multiple guides at that point, maybe one for tests, and one for code, or some other way of splitting it up.

Reply:
OK. Let's leave it here for now. If we adopt the pattern more widely, we might want to rethink where this information goes